docs/application/dotnet/tutorials/certificates/managing-certificate-profile.md

```diff
@@ -0,0 +1,83 @@
+# Managing Certificate Profile
+
+Before installing your application on a device or submitting it to the Tizen Store, it must be signed with a certificate profile.
+The signature verifies the source of the application and makes sure it has not been tampered with since its publication.
+A certificate profile is a combination of the certificates used for signing.
+
+## General Information
+
+In the menu bar, click `Tools > Tizen > Tizen Certificate Manager` to launch the Certificate Manager.
+
+![Tizen Certificate Manager with two profile](media/certification_guide18.png)
+
+* The Active profile is indicated by the check mark.
+* All of the application in this SDK will be signed with this active profile.
+* You can create, delete, and set a profile to active here.
+* Lower window shows the information of each certificate.
+* You can replace the certificate with another existing one.
+* The Tizen platform allows multiple distributor certificates up to 2 which supports other device manufacturers or telecommunication companies but it is not needed for most of developers generally.
+
+## Changing Only a Distributor Certificate in a Profile
+
+There might be an instance where you need to register more devices or change the devices to the distributor certificate.  
+However author certificates rarely needs to be changed.  
+This section describes how to do it.
+
+Do the same steps to create a certificate profile described in Creating Certificates before step [Create new or select old profile](creating-certificates.md#4-create-new-or-select-old-profile)
+
+### 1. Select an existing certificate profile
+
+Click `Select an existing certificate profile` and select a profile which you want to change the registered device then click `Next`.
+
+![Select an existing certificate profile](media/certification_guide19.png)
+
+### 2. Use the existing author certificate
+
+Click `No` to use the existing author certificate then it moves to [Create new or select existing distributor certiicate](creating-certificates.md#1-create-new-or-select-existing-distributor-certiicate) to register DUID.
```

Comment:
Change `No` to **No**.
Can we rephrase as:
Click **No** in the dialog box, which is regarding whether to create a new author certificate and moves to next step that is to [create new or select existing distributor certificate](creating-certificates.md#1-create-new-or-select-existing-distributor-certificate) to register DUID.

Need your opinion. 
Please verify whether the technical meaning is altered.

Reply:
No case means to use existing author certificate. Your rephase has the opposite meaning.

So I keep corrent sentence.